website/source/docs/enterprise/runs/variables-and-configuration.html.md

```diff
@@ -75,9 +90,44 @@ For any of the `GITHUB_` attributes, the value of the environment variable will
 be the empty string (`""`) if the resource is not connected to GitHub or if the
 resource was created outside of GitHub (like using `terraform push`).
 
+### Personal Variables
+
+Personal variables can be created at the Environment or Organization level. All
+Personal variables are private and scoped to the user that created them. Just
+like shared Environment variables, they are injected into the virtual 
+environment during the `plan` and `apply` phases.
+
+Both Personal Environment and Personal Organization variables can be used to
+override Environment variables on a per-user basis. 
+
+## Variable Hierarchy 
+
+It is possible to create the same variable in multiple places for more granular
+control. Variables are applied in the following order:
+
+1. Environment
```

Comment:
An "Environment Variable" is literally an env var, like `ATLAS_TOKEN`. They are exported in the shell during the Terraform CLI run (`terraform plan`/`terraform apply`). I don't think they necessarily fit into any hierarchy at all. I feel like we might be overloading "Environment" here. In the UI we have these things:

* Terraform Variables
* Environment Variables
* Personal Environment

I think the only hierarchy that matters here is that Personal Environment > Terraform Variables. Environment variables are their own thing.

That said I think we need to re-think the naming around some of these things. It is very confusing, even coming from someone who works on the product full-time.